apps/app/pages/index.tsx

```diff
@@ -21,6 +21,12 @@ function Dashboard() {
   const [ninjas, setNinjas] = useState([]);
   const { data: events, isLoading: isLoadingEvents } = useEvents();
   const { data: badges, isLoading: isLoadingBadges } = useBadges();
+  const [emails_sucess, setEmails_sucess] = useState([]);
```

Comment:
```suggestion
  const [sucessEmails, setSucessEmails] = useState([]);
```

Try to follow the [JS naming convention.](https://pt.wikipedia.org/wiki/CamelCase) 🙏🏼